### DISCUSSION TITLE
KNX Alexa not available

### DISCUSSION THREAD

**Discussion by SpeedyBlade** [arendst/Tasmota#22026] [Q&A]
Hello

why is Alexa not available when KNX Tasmota was flashed?

br

**Comment by s-hadinger** [comment]
I suppose that you are talking about `tasmota-knx.bin`, which indeeds has Alexa Hue disabled.

I'm not sure of the original reason, but most certainly to make the firmware small enough. Full Hue emulation takes 16KB of Flash.

You can compile your own and enable both KNX and Alexa Hue emulation

**Comment by SpeedyBlade** [comment]
yes...you are fully right ;)

But i am not able to compile my own .bin file that i can also enable both (KNX and Alexa Hue emulation).

**Comment by s-hadinger** [comment]
There are many simple options to compile yourself, like TasmoCompiler 

https://tasmota.github.io/docs/Compile-your-build/#online-compilers

**Comment by SpeedyBlade** [comment]
it would be better, that the orignal tasmota-knx.bin will be updated and give Alexa Hue emulation as an option ;)

**Comment by s-hadinger** [comment]
Option would mean yet another firmware to the list. It's easy enough with TasmoCompiler
